src/app/features/tech-record/components/tech-record-generate-letter/tech-record-generate-letter.component.ts

```diff
@@ -45,8 +44,8 @@ export class GenerateLetterComponent implements OnInit {
 
   get reasons(): Array<FormNodeOption<string>> {
     return [
-      { label: 'Authorised', value: Letters.LetterIssueTypeEnum.Authorised },
-      { label: 'Rejected', value: Letters.LetterIssueTypeEnum.Rejected }
+      { label: 'Authorized', value: LettersOfAuth.LetterTypeEnum.Authorization },
```

Comment:
Can we change this back to U.K. spelling please?

```suggestion
      { label: 'Authorised', value: LettersOfAuth.LetterTypeEnum.Authorisation },
```

Reply:
I've changed the label but the other spelling is generated